include/SQLiteCpp/Backup.h

```diff
@@ -114,14 +112,19 @@ class Backup
     int executeStep(const int aNumPage = -1);
 
     /// Return the number of source pages still to be backed up as of the most recent call to executeStep().
-    int getRemainingPageCount();
+    int getRemainingPageCount() const;
 
     /// Return the total number of pages in the source database as of the most recent call to executeStep().
-    int getTotalPageCount();
+    int getTotalPageCount() const;
 
 private:
-    // TODO: use std::unique_ptr with a custom deleter to call sqlite3_backup_finish()
-    sqlite3_backup* mpSQLiteBackup = nullptr;   ///< Pointer to SQLite Database Backup Handle
+    // Deleter functor to use with smart pointers to close the SQLite database backup in an RAII fashion.
+    struct Deleter
```

Comment:
I am sorry that I misled you with this old TODO comment; there is no use of a custom deleted when manipulating a C++ class.
Could you please revert this and restore the destructor?